client/blocks/importer/wordpress/upgrade-plan/upgrade-plan-details.tsx

```diff
@@ -14,27 +14,22 @@ import { useHasEnTranslation } from '@automattic/i18n-utils';
 import { Title } from '@automattic/onboarding';
 import { Plans2023Tooltip, useManageTooltipToggle } from '@automattic/plans-grid-next';
 import clsx from 'clsx';
-import { useTranslate } from 'i18n-calypso';
-import React, { useState, useEffect } from 'react';
+import { TranslateResult, useTranslate } from 'i18n-calypso';
+import React, { useState, useEffect, PropsWithChildren } from 'react';
```

Comment:
I think we can use type imports for `TranslateResult` and `PropsWithChildren`.
```suggestion
import { type TranslateResult, useTranslate } from 'i18n-calypso';
import React, { useState, useEffect, type PropsWithChildren } from 'react';
```